simplemenu/output/Bittboy/scripts/autoexec.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
-if [ -d "/mnt/.simplemenu" ]; then
-  echo "Cleaning .simplemenu folder"
-  find /mnt/.simplemenu -name '._*' -exec rm -v {} \;
+if find /mnt/.simplemenu -name '._*' -print -quit >/dev/null; then
```

Comment:
The same here, you are trying to prevent running the `find+delete` for windows users, but you do that by running the same `find` command part in the condition expression. So if there are no ._ files, there is no boot time difference from the previous code. 
But if there are ._ files, you add some small (probably insignificant) time to the boot by running the find again from scratch on a subsequent line...

The condition I added checking it the dir exists is meaningless in context of simplemenu executable, because these two things are dependent on each other. It's rather a safeguard for tinkerers, as if I manually change `autoexec.sh` to run other launcher, and `.simplemenu` folder doesn't exist, you get an error and not boot....

Reply:
Nope, not in the shell V90 is running. There you get an infinite loop which prevents the boot...
![IMG_6897](https://github.com/taichikuji/simplemenu/assets/113237801/6b420d78-358e-46c9-9fd4-29cb04c137f3)

Reply:
Oh, that's not very good then, I will change it to the option you suggested initially, then.